tasks/client.yml

```diff
@@ -1,9 +1,8 @@
 ---
-
 - name: download minio client
   get_url:
     url: "{{ minio_client_download_url }}"
     dest: "{{ minio_client_bin }}"
-    owner: "{{ minio_user }}"
-    group: "{{ minio_group }}"
+    owner: "root"
```

Comment:
This was done as a kind of "security measure".
Just not to download binaries from the internet as "root".
But I am not sure it still makes sense.
Thoughts?

Reply:
It isn't good to run any binary as a user who can edit this binary. We have a sane default for `minio_client_download_url` which is an official source, and we are using HTTPS so there shouldn't be any problem with tampering the file while transmitting to server.
I would also consider downloading binary once to local machine, validating checksum and then propagating it to target hosts. This increases security and reduces needed network bandwidth when used with more than 1 target. We are doing the same with most of the roles in @cloudalchemy :)